### PR TITLE
Set ExtensionInstallationFolder #2

### PR DESCRIPTION
**Pull request by grokys**
To make sure that the extension is always installed to the same directory (VS 2017 only). #990 did this originally but that wasn't merged as I thought that our issue had been resolved without it. Seems it's still needed as it's causing problems with the VS Installer.

### CODE REVIEW

**Comment by shana** [comment]
Heh, actually no it's not good, you need to set the flag on all build types, not just debug.

**Comment by grokys** [comment]
FIxed!

File: src/GitHub.StartPage/GitHub.StartPage.csproj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.23-pre\build\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.props" Condition="'$(VisualStudioVersion)' == '14.0' And Exists('..\..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.23-pre\build\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.props')" />
+  <Import Project="..\..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.215\build\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.props" Condition="Exists('..\..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.215\build\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.props')" />
```
Comment (jcansdale) [inline src/GitHub.StartPage/GitHub.StartPage.csproj]: We'll need to keep the `$(VisualStudioVersion)' == '14.0'`, if we want it to work in VS2015 and VS2017.

File: src/GitHub.VisualStudio/GitHub.VisualStudio.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="14.0">
+  <Import Project="..\..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.215\build\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.props" Condition="Exists('..\..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.215\build\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.props')" />
```
Comment (jcansdale) [inline src/GitHub.VisualStudio/GitHub.VisualStudio.csproj]: Again, we need to keep `$(VisualStudioVersion)' == '14.0'`. 😉

File: src/GitHub.VisualStudio/GitHub.VisualStudio.csproj [hunk 3]
```diff
@@ -15,6 +15,7 @@
     <OutputPath>..\..\build\$(Configuration)\</OutputPath>
     <VsixType>v3</VsixType>
     <IsProductComponent>false</IsProductComponent>
+    <ExtensionInstallationFolder>GitHub Extension for Visual Studio</ExtensionInstallationFolder>
```
Comment (jcansdale) [inline src/GitHub.VisualStudio/GitHub.VisualStudio.csproj]: How about just `GitHub`? It's in the `Extesnions` folder of Visual Studio after all. 😉

File: src/GitHub.VisualStudio/GitHub.VisualStudio.csproj [hunk 4]
```diff
@@ -700,12 +701,12 @@
...
   </Target>
   <Import Project="..\..\packages\SQLitePCL.raw_basic.0.7.3.0-vs2012\build\net45\SQLitePCL.raw_basic.targets" Condition="Exists('..\..\packages\SQLitePCL.raw_basic.0.7.3.0-vs2012\build\net45\SQLitePCL.raw_basic.targets')" />
-  <Import Project="..\..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.23-pre\build\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.targets" Condition="'$(VisualStudioVersion)' == '14.0' And Exists('..\..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.23-pre\build\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.targets')" />
   <Import Project="..\..\packages\Microsoft.VSSDK.BuildTools.15.0.26201\build\Microsoft.VSSDK.BuildTools.targets" Condition="'$(VisualStudioVersion)' == '15.0' And Exists('..\..\packages\Microsoft.VSSDK.BuildTools.15.0.26201\build\Microsoft.VSSDK.BuildTools.targets')" />
+  <Import Project="..\..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.215\build\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.targets" Condition="Exists('..\..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.215\build\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.targets')" />
```
Comment (jcansdale) [inline src/GitHub.VisualStudio/GitHub.VisualStudio.csproj]: More `$(VisualStudioVersion)' == '14.0'`...

**Comment by jcansdale** [comment]
## Visual Studio 2015

When installed from Visual Studio (F5):

image

When installed with `AllUsers=false`:

image

When installed with `AllUsers=true`:

image

## Visual Studio 2017

When installed with `AllUsers=true`:

image

Tried to revert (there's no uninstall option):
image

I'm now stuck on the following when I try to open the Extensions manager:

image

**Comment by jcansdale** [comment]
I'm going to follow instructions from #1206 to try to unscramble this.

Will this happen to anyone who has been auto-updated to the previous version?

**Comment by jcansdale** [comment]
I've followed the recovery steps from #1206.

When I loaded VS 2017 again I got the following (which is one of our old packages):

image

When I try opening the `Extensions and Updates...`, I'm still seeing:

image

😭

**Comment by jcansdale** [comment]
When launching from VS 2017 using F5, I get the following:
image

This is an easy fix. I'll push...

When it runs, assemblies are loaded form the following locations:

**Pull request merged**